### PR TITLE
release notes draft added

### PR DESCRIPTION
**Pull request by swethakundaram**
Including everything from the 10.3 issues with ticket numbers for easy reference. We can filter from here if they are not relevant. Particularly not sure about the DevOps tickets.

### CODE REVIEW

**Comment by prasanth-reddy-wavemaker-com** [comment]
Release notes for 10.3 :https://pr-139.dpwfbpxczhwpr.amplifyapp.com/learn/wavemaker-release-notes/v10-3-0

**Comment by prasanth-reddy-wavemaker-com** [comment]
@swethakundaram can you include `form-field-validator` in this commit?

File: learn/app-development/widgets/datalive/form/form-field-validator.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Validator Function"
+id: ""
+sidebar_label: "Field Validations"
+---
+---
+
+Apply validations to a single field and multiple fields using the specified methods. These methods on the form field can be used to set sync validations on a field. These validations can include default validators like required, min value, max value, or add more customizations to the basic validations.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/form-field-validator.md]: The setup has to be little better. Start by recounting when this feature should be used

File: learn/app-development/widgets/datalive/form/form-field-validator.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+- `setValidators`
+- `observeOn`
+- `setAsyncValidators`
+
+## setValidators
+
+For setting default validations for Form fields, you can use the `setValidator()` method. This method accepts an array of:
+
+- Objects for default validators
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/form-field-validator.md]: Instead say that WaveMaker ship a bunch of validators and these can be referenced and used in the setValidator method

Comment (swethakundaram) [reply learn/app-development/widgets/datalive/form/form-field-validator.md]: Updated the doc.

**Pull request merged**